spec/controllers/facility_order_details_controller_spec.rb

```diff
@@ -9,10 +9,8 @@
   before(:all) { create_users }
 
   before(:each) do
-    @authable = FactoryBot.create(:facility)
-    @facility_account = FactoryBot.create(:facility_account, facility: @authable)
+    @authable = FactoryBot.create(:setup_facility)
```

Comment:
Since we’re touching this here, I’d love to see the ivar renamed to `@facility` for better comprehension in the rest of this file, but that doesn’t have to happen in this PR.

Reply:
The `controller_test_helper` methods like `it_should_allow` that are used throughout the codebase rely on it being called `@authable` (it does look like it allows the non-ivar version `authable` as well). I agree with your intuition, but that's a bigger change. Though we could call it `facility` throughout this file and have a `@authable = facility` for the sake of the helpers. Let's look into those kind of changes the next time we're in these specs.